src/main/resources/resources/logisim/strings/gui/gui_pt.properties

```diff
@@ -332,6 +332,11 @@ editPasteItem = Colar
 editRedoItem = Refazer %s
 editSelectAllItem = Selecionar tudo
 editUndoItem = Desfazer %s
+editUndoHistoryMenu=Histórico de Desfazer
+editRedoHistoryMenu=Histórico de Refazer
+editClearHistoryAction=Limpar Histórico de Desfazer/Refazer
```

Comment:
Use lower case for "Desfazer" and "Refazer" to align with the casing of the other menu entries.

"Desfazer" -> "desfazer"
"Refazer" -> "refazer"